package.xml

```diff
@@ -62,77 +62,59 @@
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
-### Fixes across all extensions
-- Handle Apache graceful restarts more accurately #2483
-- Mark config as immutable and interned to avoid refcounting race conditions #2516
-
 ## Tracer
-This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
 
-Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+Windows support is officially GA (generally available) and considered stable.
+
+Some performance work has been undertaken to primarily reduce the overhead of telemetry reporting.
+The next release will be version 1.0 and remove deprecated and some unused code and APIs.
```

Comment:
💭 Should we also emphasize the newly added FrankedPHP Support?

Reply:
Let's not say the next release but maybe "we will soon release...". We can add thé branch for people that are curious

Reply:
Let's make sure we close the swoole(laravel octane) issue and FRs on Tuesday

Reply:
As we said we need one month between com and release, that's a possibility.
That's said we could issue a patch release if you want to merge, as indeed that's going to be easier. I'd ask the other teams if they are ok with it. And put Tushar in the loop for visibility